build.gradle

```diff
@@ -90,6 +90,7 @@ configurations.all {
 dependencyCheck {
     failBuildOnCVSS = 4
     failOnError = true
+    cveValidForHours = 24
```

Comment:
❓Is this setting useless for CI builds and just added to improve local builds? Or, is there something from CI build to CI build that allows this setting to work?